src/expr.h

```diff
@@ -56,6 +62,9 @@ class VarUseExpr : public Expr {
         UB propagate_value () { return NoUB; }
 };
 
+// Assignment expression assigns one expression to another.
+// It also converts types (replaces implicit casts with TypeCastExpr node) and updates value.
```

Comment:
Class can't "convert" (or do any other verb). Some member function can. Class description needs to explain what it represents - does it assume implicit convertsions or not. If no, you may say that it inserts explicit conversions during construction, for example.